backend/inspirehep/records/api/literature.py

```diff
@@ -171,6 +172,26 @@ def update(self, data, disable_orcid_push=False, *args, **kwargs):
                 push_to_orcid(self)
             self.push_authors_phonetic_blocks_to_redis()
 
+    def get_modified_authors(self):
+        previous_authors = get_value(self._previous_version, "authors", [])
+        previous_authors_by_uuid = {
+            author["uuid"]: author for author in previous_authors
+        }
+
+        current_authors = get_value(self, "authors", [])
+        current_authors_by_uuid = {author["uuid"]: author for author in current_authors}
+
+        for author_uuid in current_authors_by_uuid.keys():
+            if not previous_authors_by_uuid.get(author_uuid):
+                yield current_authors_by_uuid[author_uuid]
+            else:
+                if DeepDiff(
+                    current_authors_by_uuid[author_uuid],
+                    previous_authors_by_uuid[author_uuid],
+                    ignore_order=True,
+                ):
```

Comment:
why the `DeepDiff`? why can't you just do `if current_author != previous_author`?